TYPO3-Solr/ext-solr
# [BUGFIX] Fix indexing of access protected pages

# What this pr does

Fixes indexing of access protected pages and/or page contents.

# How to test

Configure access protected pages/page contents and check indexing, e.g.
- access protected pages
- access protected contents on public pages
- mix of protected contents and hide at login contents.

Resolves: #3755
